src/components/20-molecules/top-content-bar/index.js

```diff
@@ -104,9 +106,11 @@ class AXATopContentBar extends LitElement {
       <article class="m-top-content-bar">
         <div class="m-top-content-bar__container ${classMap(classes)}">
           <axa-container class="m-top-content-bar__container-component">
-            <span><slot></slot></span>
+          <div class="m-top-content-bar__children">
+            <slot></slot>
             ${btnHtml}
           </axa-container>
+          </div>
```

Comment:
You are closing axa-container before the div. You have to switch lines 112 and 113.